static/js/public/preview.js

```diff
@@ -0,0 +1,300 @@
+import Remarkable from "remarkable";
```

Comment:
I'm not sure if `/public` is a good place for it.

Yes, preview is called on a "public" details page, but it is only used by publishers. We are adding unnecessary code (not only ours but also markdown library) to a public JS file where it will not be needed at all for most of the visitors.

Possibly a `publisher` bundle would be a better place for this.
If user is previewing they are already signed in to publisher part, so this file is already downloaded. We just need to include it on details page (only in preview mode).

Reply:
Good point. I originally did it as part of the publisher bundle, and moved it for reasons I don't remember... The issue with testing perhaps? I'll move it back and see what happens. We might need to pair on getting jest working with *swiper* though @bartaz, I hit a wall.